### DISCUSSION TITLE
How to set root page("/~") conditionally based on logged user permissions

### DISCUSSION THREAD

**Discussion by ghost** [serenity-is/Serenity#6328] [Help] [answered]
Hey, I have a page "PageA" which is the root page that requires user to have a certain permission. Now, I don't wanna return an error page to users that don't have this permission but rather another valid page called "PageB". 

Where would I configure this conditional redirect? Permission checking is done before the controller is reached so I can't do it there.

**Comment by minhhungit** [comment, marked as answer]
you can use Authorization.HasPermission() to check permission and redirect to page you want
